state_transfer/example_test.go

```diff
@@ -0,0 +1,105 @@
+package state_transfer_test
+
+import (
+	"context"
+	"log"
+	"time"
+
+	"github.com/konveyor/crane-lib/state_transfer"
+	"github.com/konveyor/crane-lib/state_transfer/endpoint"
+	"github.com/konveyor/crane-lib/state_transfer/endpoint/route"
+	"github.com/konveyor/crane-lib/state_transfer/transfer"
+	"github.com/konveyor/crane-lib/state_transfer/transfer/rclone"
+	"github.com/konveyor/crane-lib/state_transfer/transport"
+	"github.com/konveyor/crane-lib/state_transfer/transport/stunnel"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/util/wait"
+	"k8s.io/client-go/rest"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+// This example shows how to wire up the components of the lib to
+// transfer data from one PVC to another
+func Example_basicTransfer() {
+	srcCfg := &rest.Config{}
```

Comment:
make these vars at the top of the file IMO

Reply:
moved